### ISSUE TITLE
testrail.getCases call return back only array with 250 values max

### ISSUE THREAD

**Issue by hatufacci** [PeterNgTr/codeceptjs-testrail#57]
According to https://www.gurock.com/testrail/docs/api/reference/cases#getcases
As of February 26, 2021 the data structure returned by bulk GET API endpoints will change. These bulk endpoints will no longer return an array of all entities, but will instead return an object with additional pagination fields and an array of up to 250 entities.
```
testrail.getCases(config.projectId, config.suiteId).then(res => {
				if (res.length) {
					validResults = allResults.filter(result => res.find(tag => tag.id == result.case_id));
					const missingLabels = allResults.filter(result => !validResults.find(vResult => vResult.case_id == result.case_id));
					if (missingLabels.length) {
						output.error(`Error: some labels are missing from the test run and the results were not send through: ${JSON.stringify(missingLabels.map(l => l.case_id))}`);
					}
				}
```
if your test rail has more than 250 tests pack of tests will be not Valid by this method and not updated in test rail, and valid cases will be in const missingLabels

**Comment by vladbalanescu** [comment]
@hatufacci Understood. Ok, I think it makes sense to have this change in I guess. 

Since I don't have the latest TestRail version - i.e. no way to test my changes - please submit your changes in a PR and I will happily review and merge. Please also be careful to put some sort of logic to say: "if TR version < X, etc", as we don't want to break consumers as myself for instance on previous TR versions.

FYI @PeterNgTr

**Comment by hatufacci** [comment]
@vladbalanescu Do you know how to check TR version via api call? I do not see such call in api

**Comment by hatufacci** [comment]
https://github.com/PeterNgTr/codeceptjs-testrail/pull/59

**Comment by PeterNgTr** [comment]
Fixed with 1.8.12
